c/dao.c

```diff
@@ -0,0 +1,290 @@
+#include "ckb_syscalls.h"
```

Comment:
Is this script only verify that withdrawing does not issue extra CKB?

Reply:
Yes that's all it does. Right now this script is included as a type script, signature verification is done as usual via a separate lock